nextflow/conf/01_optional_autometa.config

```diff
@@ -0,0 +1,37 @@
+/*
+ * -------------------------------------------------
+ *  Autometa-specific config file
+ * -------------------------------------------------
+ */
+
+// Parameters should be documented in nextflow_schema.json
```

Comment:
@Sidduppal Since there's a lot of parameters I'm thinking that maybe https://github.com/Sidduppal/Autometa/blob/documentation/docs/source/running-autometa.rst#additional-autometa-parameters should have it's own separate .rst?


I'm also thinking that at the top of said file/section we describe the best practice on documenting new parameters. 

For the runtime help documentation:

To prevent errors in the JSON, the JSON file shouldn't be modified directly- the description and help text should be entered from the nf-core GUI by 
1. going to the to top directory of Autometa
2. running ` nf-core schema build .`
3. Entering the information in the resulting GUI

Just an FYI on how this looks in the JSON: Short description is in the JSON object `description` and long description is in the JSON object for `help_text`

https://github.com/KwanLab/Autometa/blob/343d093da6e9d02a0871d92516294bba303b1d68/nextflow_schema.json#L17-L23